apps/jobs/views.py

```diff
@@ -322,6 +322,40 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
         )
 
 
+@api_view(["DELETE"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def challenge_submission_delete(request, challenge_pk, challenge_phase_pk, submission_pk):
+    """
+    Delete submission by the challenge Host
+    """
+    # check if the challenge exists or not
```

Comment:
change `check if the challenge phase exists or not` to `Check if the challenge phase exists or not`